components/gc-subway/index-en.html

```diff
@@ -27,56 +21,20 @@ <h1 property="name" id="wb-cont" class="gc-thickline">GC Subway Index</h1>
 	sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
 	est laborum.</p>
 
-<nav class="provisional gc-subway">
+<nav class="gc-subway">
 	<h2>Sections</h2>
 	<dl>
-		<dt>
-			<a href="gc-subway-en.html">
-				[Page 1]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
-		<dt>
-			<a href="page2-en.html">
-				[Page 2]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
-		<dt>
-			<a href="#">
-				[Page 3]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
-		<dt>
-			<a href="#">
-				[Page 4]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
-		<dt>
-			<a href="#">
-				[Page 5]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
-		<dt>
-			<a href="#">
-				[Page 6]
-			</a>
-		</dt>
-		<dd>
-			Page description. Lorem ipsum dolor sit amet, consectetur adipiscing elit
-		</dd>
+		<dt><a href="gc-subway-en.html">[Page 1]</a></dt>
```

Comment:
Those are broken links, please link to different kinds of examples and add description to help support the link.